fleetshard/pkg/central/reconciler/reconciler_test.go

```diff
@@ -146,7 +149,7 @@ func TestReconcileCreateWithManagedDBNoCredentials(t *testing.T) {
 			ManagedDBEnabled: true})
 
 	_, err = r.Reconcile(context.TODO(), simpleManagedCentral)
-	require.ErrorContains(t, err, "InvalidClientTokenId")
+	require.ErrorContains(t, err, "InvalidIdentityToken")
```

Comment:
Let's use go types instead of comparing string error messages:
```suggestion
	var awsErr awserr.Error
	require.ErrorAs(t, err, &awsErr)
	assert.Equal(t, awsErr.Code(), sts.ErrCodeInvalidIdentityTokenException)
```

Reply:
Done, thanks